alea/utils.py

```diff
@@ -15,15 +15,18 @@ def get_analysis_space(analysis_space: dict) -> list:
 
     for element in analysis_space:
         for key, value in element.items():
-            if value.startswith("np."):
+            if isinstance(value, str) and value.startswith("np."):
                 eval_element = (key, eval(value))
+            elif isinstance(value, str):
+                eval_element = (key,
```

Comment:
Here I also prefer the hanging indentation, like at the main branch.

Reply:
Since it's like this in the main I'll agree to this one. It does look more readable!